src/test/resources/pages/studentFeedbackSubmitPageGracePeriod.html

```diff
@@ -66,7 +66,7 @@
               </label>
               <div class="col-sm-10">
                 <p class="form-control-static">
-                  ${datetime.now}
+                  ${datetime.now} UTC+0800
```

Comment:
@amarlearning If you take a look at the history of this file: It's always been `${datetime.now}` Not sure why this is happening. Let me look into this once again.